pages/linux/inotifywait.md

```diff
@@ -26,3 +26,7 @@
 - Only watch for file modification events:
 
 `while inotifywait --event {{modify}} {{path/to/file}}; do {{command}}; done`
+
+- Continuously watch the file without exiting when an event occurs:
+
+`while inotifywait --monitor {{path/to/file}}; do {{command}}; done`
```

Comment:
I assume `{{command}}` is just some kind of dummy command, right? Could we give a specific example?

Reply:
could be anything

the most generic command I can think of is `echo 1`

Reply:
We should probably give a specific example here. Otherwise users may be confused what they should use for `{{command}}` (at least I was). How about
```suggestion
`while inotifywait --monitor {{path/to/file}}; do sleep 0; done`
```

Reply:
```suggestion
`while inotifywait --monitor {{path/to/file}}; do {{sleep 5}}; done`
```

sleep 0 is nothing.

Reply:
But isn't the use of `command` to do exactly nothing? I assumed it was kind of an `noop` command.

Reply:
I think `command` is ok, otherwise `echo` with a text is more explicit.

Reply:
I think, in the context of what this command is used for, a command which gives an output or does something to the system would be best, 

echo 1 is my best suggestion.

Reply:
I now tested this command and it works completely fine without the while loop. The loop doesn't make any difference.

In fact, `{{command}}` is never executed, because `inotifywait --monitor file` never returns a value, because it doesn't exit on its own.

Reply:
Could you provide a suggestion message

Reply:
Im a little unsure what you would like to change, so if we merge this, would you be willing to make the changes and the PR?

Reply:
Sure!